packages/e2e-tests/fixtures/blocks/core__embed.json

```diff
@@ -6,7 +6,9 @@
         "attributes": {
             "url": "https://example.com/",
             "caption": "Embedded content from an example URL",
-            "allowResponsive": true
+			"allowResponsive": true,
+			"responsive": false,
+			"previewable": true
```

Comment:
Weird, why is the indentation different?

Reply:
It had spaces so I changed to `tabs`